plugins/webp-uploads/readme.txt

```diff
@@ -1,15 +1,15 @@
-=== WebP Uploads ===
+=== Modern Image Formats ===
 
 Contributors:      wordpressdotorg
 Requires at least: 6.4
 Tested up to:      6.5
 Requires PHP:      7.0
-Stable tag:        1.0.6
+Stable tag:        1.1.0
 License:           GPLv2 or later
 License URI:       https://www.gnu.org/licenses/gpl-2.0.html
 Tags:              performance, images, webp
 
-Creates WebP versions for new JPEG image uploads if supported by the server.
+Converts images to more modern formats such as WebP or AVIF during upload.
 
 == Description ==
```

Comment:
Ditto above about mentioning the old name.

Reply:
Done in https://github.com/WordPress/performance/pull/1101/commits/0c04a92f8cdf1eb4067a0ca11f2c0b92ee4a7b24